src/core/AnimatedValue.js

```diff
@@ -29,6 +47,11 @@ export default class AnimatedValue extends AnimatedNode {
     this.__forceUpdateCache(value);
   }
 
+  setValue(value) {
+    this.__detachAnimation();
```

Comment:
I think when the value is set using imperative API the animation should stop. It seems like calling into "detach" would not actually stop the animation.

Reply:
It actually stops animation 

Please see line 27